sirepo/srschema.py

```diff
@@ -206,6 +207,20 @@ def _validate_enum(val, sch_field_info, sch_enums):
         raise AssertionError(util.err(sch_enums, 'enum {} value {} not in schema', type, val))
 
 
+def _validate_job_run_mode(field_name, schema):
+    if field_name != 'jobRunMode':
+        return
+    from sirepo import pkcli
+    t = schema.simulationType
+    m = pkcli.import_module(t)
+    if hasattr(m, 'run_background'):
+        raise AssertionError(
+            f'simulation_type={t} cannot have'
+            ' pkcli.run_background because it supports slurm. Slurm only'
```

Comment:
Pet peeve: the idea of concatenating strings without an operator in between is problematic. I know this is valid Python, but it seems to me, it's also strange. Rather, just add a leading plus to the continued strings.

Reply:
f13e9db85